content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -314,6 +314,57 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl using package management
+{{< tabs name="other_kubectl_uninstall" >}}
+{{% tab name="apt-get" %}}
+If you are on Ubuntu or another Debian based Linux distribution
+
+```shell
+sudo apt-get purge kubectl
+sudo apt-get autoremove
+```
+
+The executable binaries are stored in the ```/usr/bin or usr/local/bin```, library files in ```usr/lib``` and configuration files in ```/etc``` 
+It is recommended to manually remove the library files and static data before uninstalling the ```kubectl``` from the system
+
+{{% /tab %}}
+
+{{% tab name="curl" %}}
+If you have used curl to install kubectl, then the uinstallation process can be done using :-
+
+```shell
+sudo rm -rf /usr/local/bin/kubectl
+sudo rm -rf /usr/local/lib/kubectl
```

Comment:
Hi @infinite-binaryloop, you can mention here it as - 

```bash
sudo rm $(which kubectl)
```
which will remove it at first time.

Reply:
Also, Above it is mentioned that kubectl is installed in /usr/local/bin/ .

Reply:
Hey @utkarsh-singh1 
Thank you for the comment. 
Yes, you are right! we can use the above mentioned command, that is 
`sudo rm $(which kubectl)`.
It will be removing the package from the `/usr/bin/`. We can certainly do that.